rootfs/usr/share/inputplumber/devices/50-steam_deck.yaml

```diff
@@ -24,14 +24,30 @@ matches:
 # One or more source devices to combine into a single virtual device. The events
 # from these devices will be watched and translated according to the key map.
 source_devices:
+  # Gamepad
   - group: gamepad
     hidraw:
       vendor_id: 0x28de
       product_id: 0x1205
       interface_num: 2
+  # Touchscreen
+  - group: mouse
+    hidraw:
+      vendor_id: 0x2808
+      product_id: 0x1015
+  - group: mouse
+    blocked: true
+    evdev:
+      name: "FTS3528:00 2808:1015*"
+  # Keyboard
+  - group: keyboard
+    evdev:
+      name: AT Translated Set 2 keyboard
+      phys_path: isa0060/serio0/input0
 
 # The target input device(s) that the virtual device profile can use
 target_devices:
   - gamepad
   - mouse
   - keyboard
+  - touchscreen
```

Comment:
I think we should be more specific as to the model here. When I add the Legion Go and OrangePi Neo touchscreens, as well as evdev only touchscreens, we will need to have more than one option available. This generic name is sub-optimal when it's not universal.